dlt-database/entity/0002-refactor_add_community/ConfirmedTransaction.ts

```diff
@@ -10,7 +10,7 @@ import {
 import { Decimal } from 'decimal.js-light'
 
 import { DecimalTransformer } from '../../src/typeorm/DecimalTransformer'
-import { Account } from '../Account'
+import { Account } from './Account'
```

Comment:
the same as in the entity `Account`

Reply:
please add a comment above the code, why you make this change

Reply:
done.